crates/fuel-streams-core/src/outputs/subjects.rs

```diff
@@ -0,0 +1,143 @@
+use fuel_core_types::{fuel_tx::TxId, fuel_types::ContractId};
+use fuel_streams_macros::subject::{IntoSubject, Subject};
+
+use crate::types::*;
+
+pub const OUTPUTS_WILDCARD_LIST: &[&str] = &[
+    OutputsByIdSubject::WILDCARD,
+    OutputsAllSubject::WILDCARD,
+    OutputsCoinSubject::WILDCARD,
+    OutputsContractSubject::WILDCARD,
+    OutputsChangeSubject::WILDCARD,
+    OutputsVariableSubject::WILDCARD,
+    OutputsContractCreatedSubject::WILDCARD,
+];
+
+#[derive(Subject, Debug, Clone, Default)]
+#[subject_wildcard = "outputs.>"]
+#[subject_format = "outputs.{tx_id}.{index}.>"]
+pub struct OutputsAllSubject {
```

Comment:
This one doesn't need to exist since you can do `outputs.>` to get results for all subjects

Reply:
But for this query you can do `outputs.*.{tx_id}.{index}.>`, we don't need an extra subject just for this query

Reply:
I think it makes sense to implement this filter in the SDK. Let's proceed with it. 🙏🏻